dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -802,6 +808,15 @@ protected void rawDelete(Context context, Item item) throws AuthorizeException,
         itemDAO.delete(context, item);
     }
 
+    protected void removeRequest(Context context, Item item) throws SQLException {
+        Iterator<RequestItem> requestItems = requestItemService.findByItem(context, item);
+        while (requestItems.hasNext()) {
+            RequestItem requestItem = requestItems.next();
+            log.info("Remove Request Item: " + requestItem.getID());
```

Comment:
@jensvannerum : Can we please remove this `log.info`?  This looks like it'd only be useful for debugging & isn't useful otherwise.